index.html

```diff
@@ -586,6 +586,34 @@
   </div>
 </div>
 <!-- ends Modify speed -->
+<!-- Text Watermark -->
+<span data-toggle="modal" data-target=".text_watermark"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Create opaque centered text watermark ">Text Watermark</button></span>
+<div class="modal fade text_watermark" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="well">
+        <h3>Create centered, transparent text watermark</h3>
+        <p>E.g For creating access copies with your institutions name</p>
+        <p><code>ffmpeg -i <i>input_file</i> -vf drawtext="fontfile=<i>font_path</i>:fontsize=<i>font_size</i>:text=<i>watermark_text</i>:fontcolor=<i>font_colour</i>:alpha=0.4:x=w/2-tw/2:y=h/2-th/2" <i>output_file</i></code></p>
+        <dl>
+          <dt>ffmpeg</dt><dd>starts the command</dd>
+          <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+          <dt>-vf drawtext=</dt><dd>This calls the drawtext filters with the following options:
+          <dl>
+            <dt>fontfile=<i>font_path</i></dt><dd> Set path to font. For example in OSX: <code>"fontfile=/Library/Fonts/AppleGothic.ttf"</code></dd>
+            <dt>fontsize=<i>font_size</i></dt><dd> Set font size. <code>35</code> is a good starting point for SD. Ideally this value is proportional to video size, for example use ffprobe to acquire video height and divide by 14.</dd>
```

Comment:
I'm not certain, the docs are vague: 
https://ffmpeg.org/ffmpeg-filters.html#Syntax

    The font size to be used for drawing text. The default value of fontsize is 16.